modules/farm_fd2_examples/src/entrypoints/soil_disturbance/App.vue

```diff
@@ -0,0 +1,269 @@
+<template>
+  <h3>Soil Disturbance Example</h3>
+  <p data-cy="description">
+    The SoilDisturbance component collects data on soil disturbances.
+  </p>
+
+  <hr />
+  <SoilDisturbance
+    id="soil-disturbance"
+    data-cy="soil-disturbance"
+    v-bind:required="required"
+    v-model:area="form.area"
+    v-model:depth="form.depth"
+    v-model:equipment="form.equipment"
+    v-bind:includePasses="includePasses"
+    v-model:speed="form.speed"
+    v-model:passes="form.passes"
+    v-bind:showValidityStyling="validity.showStyling"
+    v-on:valid="
+      (valid) => {
+        validity.soilDisturbance = valid;
+      }
+    "
+    v-on:ready="createdCount++"
+  />
+  <hr />
+
+  <h5>Component Props:</h5>
+  <table class="example-table">
+    <thead>
+      <th>Prop</th>
+      <th>Control</th>
+    </thead>
+    <tbody>
+      <tr>
+        <td>required</td>
+        <td>
+          <BFormCheckbox
+            id="required-checkbox"
+            data-cy="required-checkbox"
+            switch
+            v-model="required"
+          />
+        </td>
+      </tr>
+      <tr>
+        <td>showValidityStyling</td>
+        <td>
+          <BFormCheckbox
+            id="styling-checkbox"
+            data-cy="styling-checkbox"
+            switch
+            v-model="validity.showStyling"
+          />
+        </td>
+      </tr>
+      <tr>
+        <td>Add Planter</td>
+        <td>
+          <BButton
+            id="add-equipment-button"
+            data-cy="add-equipment-button"
+            variant="outline-primary"
+            size="sm"
+            v-on:click="addEquipment"
+          >
+            Add Planter
```

Comment:
Use prop name `equipment` here for consistency.  Consider controls for other props as well.